pkg/clusterregistryconfig/flags.go

```diff
@@ -152,25 +159,30 @@ func GetClusterRegistryConfigOptions(cmd *pflag.FlagSet,
 	}
 
 	if enableRegistriesConfig && interactive.Enabled() {
-		allowedRegistriesInputs, err := interactive.GetString(interactive.Input{
-			Question: "Allowed Registries",
-			Help:     cmd.Lookup(allowedRegistriesFlag).Usage,
-			Default:  strings.Join(defaultAllowedRegistries, ","),
-		})
-		if err != nil {
-			return nil, fmt.Errorf("Expected a valid value for allowed registries: %s", err)
+		// Allowed registries and blocked registries are mutually exclusive
+		if result.blockedRegistries == nil {
+			allowedRegistriesInputs, err := interactive.GetString(interactive.Input{
+				Question: "Allowed Registries",
+				Help:     cmd.Lookup(allowedRegistriesFlag).Usage,
+				Default:  strings.Join(defaultAllowedRegistries, ","),
+			})
+			if err != nil {
+				return nil, fmt.Errorf("Expected a valid value for allowed registries: %s", err)
```

Comment:
what is a valid value? can we show an example or explain if it's a comma separated string?